utils/keyboard-shortcut.js

```diff
@@ -0,0 +1,29 @@
+export const ALT = 'alt';
+export const CONTROL = 'mod';
+export const META = 'meta';
+export const SHIFT = 'shift';
+
+export default function keyboardShortcut(...keys) {
```

Comment:
Cool! I did something similar in 

https://github.com/WordPress/gutenberg/blob/99c70d7de22d0d64f3af685256ce6e67828dee14/utils/keycodes.js#L48-L61.

Might it make sense to have two functions to retrieve meta and access combinations, since we will and probably should never have any other combination? In that case the only argument would be the letter. It also means that we and plugins are discouraged from starting to use any other combination, which will conflict with browsers and operation systems.

Reply:
So `getMetaCombination` would be `Cmd` for Mac and `Ctrl` for Windows. `getAccessCombination` would be `Ctrl+Alt` for Mac and `Shift+Alt` for Windows.

Reply:
Yeah, the access and meta abstractions make sense to me.

Reply:
Oh, cool! Sorry, I totally missed this, my bad 😓

The function I made tried to abstract this a bit but did it with Windows-specific words (based on simply marketshare of Windows machines)–so `CONTROL` is actually `'Ctrl+'` for Windows and `'⌘'` for Mac.

Your method of providing the "combination" helpers is indeed much nicer. I'll refactor this and go for that approach 👍